k8s/openebs-operator.yaml

```diff
@@ -80,6 +80,7 @@ spec:
     metadata:
       labels:
         name: maya-apiserver
+        component: apiserver
```

Comment:
How about `openebs.io/component: maya-apiserver`
In addition, please specify the other PRs that should update the charts etc in this PR message.